src/scripts/main.js

```diff
@@ -1,8 +1,19 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashboard = document.querySelector('.dashboard');
```

Comment:
createdd rows should be inside tbody
```suggestion
const dashboard = document.querySelector('tbody');
```
actual
![Screenshot from 2023-06-10 14-57-18](https://github.com/mate-academy/js_task_generate_table_DOM/assets/33029864/d49aafc9-193d-41ea-b28a-a1856554082d)